app/post/export/post-export-directive.js

```diff
@@ -0,0 +1,75 @@
+module.exports = PostExportDirective;
```

Comment:
I think this should probably still be under app/post/views - since its only used on views right now.